src/Extension/Engine/SiteTreePublishingEngine.php

```diff
@@ -160,44 +167,49 @@ public function collectChanges($context)
 
     /**
      * Execute URL deletions, enqueue URL updates.
+     * @throws ValidationException
      */
     public function flushChanges()
     {
-        $queue = static::$queueService ?? QueuedJobService::singleton();
+        $queueService = static::$queueService ?? QueuedJobService::singleton();
 
-        if (!empty($this->toUpdate)) {
-            foreach ($this->toUpdate as $queueItem) {
-                $job = Injector::inst()->create(GenerateStaticCacheJob::class);
+        if (count($this->toUpdate) > 0) {
```

Comment:
Why has this changed from `!empty()` to `count() > 0`. The latter is going to be less efficient, there's no point calculating the total length of an array just to assert it's greater than 0, this is what `empty()` is for

Reply:
This is just a different linting setup. My team generally avoid using empty but I'm ok with keeping it here as the module linting setup doesn't prevent using empty.

Reply:
From a bit of cursory research online, count takes 2× as long, even on empty arrays, so using count is regressive in terms of performance and so should never be used simply to check if arrays have elements or not.